server/lib/ga-collector.js

```diff
@@ -80,6 +80,20 @@ GACollector.prototype = {
           uid: body.uniqueUserId
         };
 
+        if (body.screen) {
+          var screen = body.screen;
+
+          // screen resolution and viewport parameter
+          // https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters?hl=en#sr
+          if (screen.width && screen.height) {
+            gaData.sr = screen.width + 'x' + screen.height;
+          }
+
+          if (screen.clientWidth && screen.clientHeight) {
+            gaData.vs = screen.clientWidth + 'x' + screen.clientHeight;
```

Comment:
In the referred to doc (L87), I don't see a `vs`, but I do see a `vp` - https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters?hl=en#vp